docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -44,18 +142,18 @@ You can find your customer UUID in Yugabyte Platform as follows:
       [http|https]://myPlatformServer/api/customers
   ```
 
-  For example:
+  <br>For example:
 
   ```sh
   curl -X "X-AUTH-YW-API-TOKEN: e5c6eb2f-7e30-4d5e-b0a2-f197b01d9f79" \
     http://localhost/api/customers
   ```
 
-- Copy your UUID from the resulting JSON output shown in the following example, only without the double quotes and square brackets:
+- Copy your UUID from the resulting JSON output shown in the following example, omitting the double quotes and square brackets:
 
   ```
   ["6553ea6d-485c-4ae8-861a-736c2c29ec46"]
   ```
 
-  To find a universe's UUID in Yugabyte Platform, click **Universes** in the left column, then click the name of the universe. The URL of the universe's **Overview** page ends with the universe's UUID. For example, `http://myPlatformServer/universes/d73833fc-0812-4a01-98f8-f4f24db76dbe`
+  <br>To find a universe's UUID in Yugabyte Platform, click **Universes** in the left column, then click the name of the universe. The URL of the universe's **Overview** page ends with the universe's UUID. For example, `http://myPlatformServer/universes/d73833fc-0812-4a01-98f8-f4f24db76dbe`
```

Comment:
This bit shouldn't be indented, it is separate from find your user UUID

Reply:
Deleting this section.